src/goDebugConfiguration.ts

```diff
@@ -29,16 +29,22 @@ export class GoDebugConfigurationProvider implements vscode.DebugConfigurationPr
 			if (!activeEditor || activeEditor.document.languageId !== 'go') {
 				return;
 			}
+			let debugMode;
+			let testFileRegExp = RegExp('.*_test\.go$', 'g');
+			if (testFileRegExp.test(activeEditor.document.fileName) === true) {

```

Comment:
isnt it simpler with `if (activeEditor.document.fileName.endsWith('_test.go'))` ?